### PR TITLE
Document the safe-to-evict annotation on the webhook deployment

### PR DESCRIPTION
**Pull request by imjasonh**
/kind documentation

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [y] [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [n/a] [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [y] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [y] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [y] Release notes block below has been filled in or deleted (only if no user facing changes)

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: docs/enabling-ha.md [hunk 1]
```diff
@@ -88,6 +88,9 @@ spec:
   minReplicas: 1
 ```
 
+By default, the Webhook deployment is configured to block a [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) from scaling down the node that's running the only replica of the deployment using the `cluster-autoscaler.kubernetes.io/safe-to-evict` annotation -- when the Webhook service is unreachable, Tekton resources can't be created, updated or deleted.
```
Comment (pierretasci) [inline docs/enabling-ha.md:91]: [nit] The `--` confused me a bit reading this for the first time. Consider making this two sentences.

**Comment by pierretasci** [comment]
/approve

Just a small syntax nit

File: docs/enabling-ha.md (hunk 1 above)
Comment (imjasonh) [reply docs/enabling-ha.md]: Good call, done :)

**Comment by sbwsg** [comment]
Switching my lgtm to an approve.

/approve

**Pull request merged**